src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -1140,21 +1140,38 @@ public static bool IsGraphQLReservedName(Entity entity, string databaseColumnNam
             };
 
             string tablePrefix = GetTablePrefix(conn.Database, schemaName);
+            string queryprefix = string.IsNullOrEmpty(tablePrefix) ? string.Empty : $"{tablePrefix}.";
```

Comment:
```suggestion
            string queryPrefix = string.IsNullOrEmpty(tablePrefix) ? string.Empty : $"{tablePrefix}.";
```

Reply:
Do we want to go a step ahead and check for `whitespace` string as well (string.isNullOrWhiteSpace())?

Reply:
I dont think GetTableprefix would ever return a whitespace? databasename is taken from the connection string and then the schemaname is also validated in input.